library/src/test/kotlin/it/krzeminski/githubactions/dsl/expressions/ContextsTest.kt

```diff
@@ -75,4 +75,25 @@ class ContextsTest : FunSpec({
             } shouldBe expr("startsWith('Let\\'s dance', 'Let\\'s dance and be happy!')")
         }
     }
+
+    test("GitHub context") {
+        assertSoftly {
+            expr { github.token } shouldBe expr("github.token")
+            expr { github.job } shouldBe expr("github.job")
+            expr { github.sha } shouldBe expr("github.sha")
+            expr { github.repository_owner } shouldBe expr("github.repository_owner")
+            expr { github.repositoryUrl } shouldBe expr("github.repositoryUrl")
+            expr { github.action } shouldBe expr("github.action")
+        }
+    }
+
+    test("GitHub.event context") {
+        assertSoftly {
+            expr { github.eventPush.ref } shouldBe expr("github.event.ref")
+            expr { github.eventPullRequest.pull_request.number } shouldBe expr("github.event.pull_request.number")
```

Comment:
Would it be possible to change the DSL to
```suggestion
            expr { github.event.pull_request.number } shouldBe expr("github.event.pull_request.number")
```
so that it's 1:1 with how it's seen in the YAML?

Reply:
In your example that would be:
`github.event.pull_request.pull_request.number`

No, not a mistake.